api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/CounterHandler.java

```diff
@@ -193,7 +193,8 @@ public void getMetric(@Suspended final AsyncResponse asyncResponse, @PathParam("
 
     @DELETE
     @Path("/{id}")
-    @ApiOperation(value = "Deletes the metric and associated data points, and updates internal indexes.")
+    @ApiOperation(value = "Deletes the metric and associated data points, and updates internal indexes."
+            + " Note: compressed data will not be deleted and still be accessible.")
```

Comment:
I would make one more minor change, something along the lines of, "Note: compressed data will not be deleted. It is instead deleted as part of the normal data expiration as defined by the data retention settings; consequently, compressed data will still be accessible until it expires."